index.html

```diff
@@ -0,0 +1,553 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        Web Page
+    </title>
+    <meta charset="utf-8">
+    <meta name='viewport' content="width=device-width" , initial-scale=1'>
```

Comment:
I guess you are missing a double quote here.  For the rest of the html I dont see any code coloring